feature/onboarding/src/main/java/com/napzak/market/onboarding/termsAgreement/TermsAgreementScreen.kt

```diff
@@ -0,0 +1,174 @@
+        onAllAgreementClick = viewModel::onAllAgreementClick,
+        onAgreementClick = viewModel::onAgreementClick,
+        onBackClick = onBackClick,
+        onNextClick = onNextClick,
+    )
+}
+
+@Composable
+fun TermsAgreementScreen(
+    uiState: TermsAgreementUiState,
+    onAllAgreementClick: () -> Unit,
+    onAgreementClick: (TermType) -> Unit,
+    onBackClick: () -> Unit,
+    onNextClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val context = LocalContext.current
+
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(NapzakMarketTheme.colors.white)
+            .padding(horizontal = 20.dp, vertical = 50.dp),
+    ) {
+        TermsAgreementTopBar(
+            onBackClick = onBackClick,
+        )
+
+        Spacer(modifier = Modifier.height(30.dp))
+
+        Text(
+            text = stringResource(onboarding_title),
+            style = NapzakMarketTheme.typography.title20b,
+            color = NapzakMarketTheme.colors.gray400,
+        )
+
+        Spacer(modifier = Modifier.height(24.dp))
+
+        NapzakCheckedButton(
+            checked = uiState.isAllAgreed,
+            text = stringResource(onboarding_all_agreement_title),
+            backgroundColor = NapzakMarketTheme.colors.gray50,
+            onCheckedChange = { onAllAgreementClick() },
+            modifier = Modifier
+                .fillMaxWidth(),
+        )
+
+        Spacer(modifier = Modifier.height(20.dp))
+
+        Column {
+            TermType.entries.forEach { term ->
+                NapzakCheckedButton(
+                    checked = uiState.isChecked(term),
+                    text = term.getDisplayLabel(),
+                    onCheckedChange = { onAgreementClick(term) },
+                    onIconClick = { context.openUrl(term.detailContentUrl) },
+                    icon = ImageVector.vectorResource(ic_arrow_right),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 8.dp),
+                )
+            }
+        }
+
+        Spacer(modifier = Modifier.weight(1f))
+
+        NapzakButton(
+            text = stringResource(onboarding_next),
+            onClick = onNextClick,
+            enabled = uiState.isAllAgreed,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 32.dp),
+        )
+    }
+}
+
+@Composable
+private fun TermsAgreementTopBar(
+    onBackClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Icon(
+            imageVector = ImageVector.vectorResource(ic_arrow_left),
+            contentDescription = null,
+            tint = Color.Unspecified,
+            modifier = Modifier
+                .clickable { onBackClick() }
```

Comment:
p1: 여기서도 람다를 없애서 파라미터로 넣어주면 됩니다!
```suggestion
                .clickable(onClick = onBackClick)
```